README.md

```diff
@@ -107,6 +107,9 @@ npm run release -- --no-verify
 standard-version --no-verify
 ```
 
+### Signing commits and tags
```

Comment:
sorry for the nit, but mind adding a space below the heading, I think this might cause issues for some markdown renderers.